state/protocol/badger/snapshot_test.go

```diff
@@ -674,7 +674,6 @@ func TestSealingSegment(t *testing.T) {
 			parent := root
 			for i := 0; i < flow.DefaultTransactionExpiry+1; i++ {
 				next := unittest.BlockWithParentProtocolState(parent)
-				next.Header.View = next.Header.Height + 1 // set view so we are still in the same epoch
```

Comment:
By default, `BlockWithParentProtocolState` creates a child block, whose view can be up to 10 bigger than its parent. If we are unlucky and randomly select unfavourable view increments, the resulting view of the last generated block might fall beyond the epoch. The comment explicitly explains this. By removing this, we are probably making the test flakey. 

I fixed this in commit https://github.com/onflow/flow-go/pull/6933/commits/54001f71af279d443e81f61c730efb67aeea76be